cumulus_library/studies/core/builder_core_medication.sql

```diff
@@ -0,0 +1,153 @@
+-- noqa: disable=all
+/*
+This is a reference output of the SQL generated by builder_core_medication.py 
+that is used by the core__encounter_type table. It is not invoked directly.
+*/
```

Comment:
Good comment, but I have a further clarifying question - is this used at all by a production run? Like, _could_ this be in a `tests/` directory, or is not invoked directly, but is still used somehow?

Reply:
This is more about documentation at @comorbidity 's request - the idea being that a researcher can get an idea what the builder is doing in the maximal case, without actually needing to understand the builder's logic. But it is never run in any form.